src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,107 @@
+const { expenseSchema } = require('../shemas');
```

Comment:
There is a typo in the import path for `expenseSchema`. It should be `../schemas` instead of `../shemas` to match the correct directory name.